velox/connectors/hive/storage_adapters/s3fs/tests/S3MultipleEndpointsTest.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/init/Init.h>
+
+#include "gtest/gtest.h"
+#include "velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.h"
+#include "velox/connectors/hive/storage_adapters/s3fs/tests/S3Test.h"
+#include "velox/exec/TableWriter.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::core;
+using namespace facebook::velox::exec;
+using namespace facebook::velox::exec::test;
+using namespace facebook::velox::connector;
+using namespace facebook::velox::connector::hive;
+using namespace facebook::velox::dwio::common;
+using namespace facebook::velox::test;
+using namespace facebook::velox::filesystems;
+
+static const std::string kConnectorId1 = "test-hive1";
```

Comment:
string_view should be used for static const globals

Reply:
The `newConnector` API needs a `const std::string`.
Clang says string_view cannot be converted to a const std::string.

Reply:
if you need an actual std::string you can always explicitly `std::string(std::string_view()))` it, but no big deal.

Reply:
fixed